tmdrv_devices/thrustmaster_tmx.py

```diff
@@ -0,0 +1,9 @@
+name = 'Thrustmaster TMX'
+idVendor = 0x044f
+idProduct = [0xb67e, 0xb65d, 0xb67f]
+control = [
+	{'step':1, 'request_type':0x41, 'request':83, 'value':0x0001, 'index':0x0000, 'data':b''},
+	{'step':2, 'request_type':0x41, 'request':83, 'value':0x0007, 'index':0x0000, 'data':b''},
+]
+jscal = '6,1,0,32505,32505,16574,2049063,1,0,519,519,1036399,1067305,1,0,512,512,1050596,1054724,1,0,511,511,1050596,1048544,1,0,0,0,536854528,536854528,1,0,0,0,536854528,536854528'
+dev_by_id = 'usb-Thrustmaster_Thrustmaster_TMX_Racing_Wheel-joystick'
```

Comment:
Additional feedback. I wanted to further test my hypothesis about the symlink. In tmdrv.py I replaced lines 49, 50 with a pass and commented 64-71 after I ran tmdrv to init the device. This essentially should only run the jscal line of code. On the second run, bypassing this usb error, the jscal data loads fine. 

A few times i tried doing a rapid succession of ls -al in the by-id directory and it doesn't appear until after the script has errored on the jscal line.